build.gradle

```diff
@@ -110,18 +107,27 @@ tasks.named('check') { check ->
   check.dependsOn(checkFixtureJars)
 }
 
-tasks.register('updateLicenses', de.undercouch.gradle.tasks.download.Download) { task ->
-  task.src('https://spdx.org/licenses/licenses.json')
-  def destDir = file('src/main/resources/app/cash/licensee')
-  task.dest(destDir)
-  task.doLast {
-    def destFile = new File(destDir, 'licenses.json')
-    def json = new JsonSlurper().parse(destFile)
-    json.licenses = json.licenses.sort { a,b -> a.licenseId <=> b.licenseId }
```

Comment:
Sorry I COMPLETELY forgot that I was doing this. Maybe we do need sort... I thought they were finally sorting because they said they would.

Reply:
Sorry I lost track of this PR.

Yeah we need to revert. It's definitely not sorted, despite me asking.